overview.md

```diff
@@ -164,10 +164,32 @@ TODO
 
 ## Documentation and code exploration
 
+### Rustdoc
+
+You probably don't need to use rustdoc directly (instead going through `cargo doc`). Accepts a crate
```

Comment:
I would start with a sentence that states what Rustdoc does - "Rustdoc generates API documentation for Rust crates," or something like that